cmd/manifest.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/open-feature/cli/internal/manifest"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+)
+
+var manifestCmd = &cobra.Command{
+	Use:   "manifest",
+	Short: "Manage OpenFeature manifests",
+	Long:  `Manage OpenFeature manifests with subcommands to init, validate, and compare manifests`,
+}
+
+type InitManifest struct {
+	Schema string `json:"$schema,omitempty"`
+	manifest.Manifest
+}
+
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize a new manifest",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if _, err := os.Stat(ManifestPath); err == nil {
+			overwrite, _ := cmd.Flags().GetBool("overwrite")
+			if !overwrite {
+				pterm.Warning.Printf("%s already exists. Use --overwrite to overwrite", ManifestPath)
+				return nil
+			}
+		}
+
+		m := &InitManifest{
+			Schema: "https://raw.githubusercontent.com/open-feature/cli/refs/heads/main/schema/v0/flag_manifest.json",
+			Manifest: manifest.Manifest{
+				Flags: map[string]any{},
```

Comment:
It's possible we could use the flag structs here in place of `any`

Reply:
I don't believe so since Go doesn't support discriminatory unions. However, I'm still new to Go, so there may be a better way.

I have a few changes locally that I'm still working on that should improve this a bit more. Hopefully I'll have it wrapped up soon.  🤞

Reply:
I'm glad I'm not alone in being newish to golang 😅 

I was thinking this might be possible with go generics or interfaces, but I do wonder if it might result in casting types ultimately which could balloon. Thanks for discussing!